README.md

```diff
@@ -27,7 +25,7 @@ After you're finished please remove all the comments and instructions!
 
 <div align="center">
   <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <img src="elim.png" alt="logo" width="140"  height="auto" />
   <br/>
```

Comment:
- [ ] This project is really well done and it should proudly wear your logo 💪 . I notice your logo is in the images folder and so to access it, you need to tweak the path a bit. Line 28 should probably look like this: 
```<img src="./images/elim.png" alt="logo" width="140" height="auto" />```

Do make the change in this very pull request and submit this pull request for review instead of opening a new pull request.